src/hooks/hooks.tsx

```diff
@@ -190,13 +177,13 @@ export const useLoadLinkedIssues = () => {
             client
               .getEntityAssociation(
                 "linkedJiraIssues",
-                context?.data.ticket.id as string,
+                context?.data?.ticket.id as string,
               )
               .delete(id),
             client
               .getEntityAssociation(
                 "linkedJiraIssues",
-                context?.data.ticket.id as string,
+                context?.data?.ticket.id as string,
```

Comment:
This doesn't make sense, optionally chain but typecast as string. This surely means the value can be a string or undefined.

Reply:
fixed